### PR TITLE
Let checkForPhpNoticesOrWarnings speak

### PR DESCRIPTION
**Pull request by muhme**
Sometimes a PHP problem message directly in the HTML code causes rendering problems and
makes the error message invisible in the browser and on the screenshot. With this change
the PHP problem message is shown.

See the change by sample. Before:
```
  Test in frontend that the related items module
    1) "after each" hook for "can display a list of related articles based on the metakey field"

  0 passing (748ms)
  1 failing

  1) Test in frontend that the related items module
       "after each" hook for "can display a list of related articles based on the metakey field":
     AssertionError: expected true to equal false

Because this error occurred during a `after each` hook we are skipping all of the remaining tests.
      at Context.eval (webpack://joomla/./node_modules/joomla-cypress/src/support.js:98:0)
```

After:
```
  Test in frontend that the related items module
    1) "after each" hook for "can display a list of related articles based on the metakey field"

  0 passing (1s)
  1 failing

  1) Test in frontend that the related items module
       "after each" hook for "can display a list of related articles based on the metakey field":
     Error: Unwanted PHP Deprecated: "  DateTime::__construct(): Passing null to parameter https://github.com/joomla-projects/joomla-cypress/pull/1 ($datetime) of type string is deprecated in <b>/joomla-cms/libraries/src/Date/Date.php</b> on line <b>126</b>"

Because this error occurred during a `after each` hook we are skipping all of the remaining tests.
      at Context.eval (webpack://joomla/./node_modules/joomla-cypress/src/support.js:116:0)
```

✅ Tested with actual Joomla brnach 4.4-dev, which has 4 Deprecated/Warnings on Intel macOS 14.5 Sonoma with PHP 8.3.8, Ubuntu 24.04 LTS Noble Numbat with PHP 8.3.6 and Windows 11 with PHP 8.1.10.

### CODE REVIEW

**Comment by muhme** [comment]
In my opinion, it is worth creating a new NPM version 1.1.1 with this commit.

File: src/support.js [hunk 1]
```diff
@@ -81,27 +81,38 @@ const supportCommands = () => {
...
+   * Looking for:
+   * - <b>Warning</b>:
+   * - <b>Deprecated</b>:
+   * - <b>Notice</b>:
+   * - <b>Strict standards</b>:
+   *
+   * @memberof Cypress.Commands
    * @method checkForPhpNoticesOrWarnings
-   * @returns Chainable
+   * @returns {Cypress.Chainable}
```
Comment (laoneo) [inline src/support.js]: This method returns nothing

Comment (muhme) [reply src/support.js]: Yes, this is correct. I have the method documentation rewritten:
```
@returns {void} - It does not return any value, but it is chainable with other Cypress commands.
```

**Comment by laoneo** [comment]
Zhanks!

**Pull request merged**